src/vs/vscode.proposed.d.ts

```diff
@@ -1100,6 +1105,11 @@ declare module 'vscode' {
 			 */
 			name: string;
 
+			/**
```

Comment:
unrelated to this PR but you could fix the typing being nested inside a namespace move it out to where it belongs

Reply:
The `createStatusBarItem` also belongs to the `window` namespace in `vscode.d.ts`. Thus we have to keep it in the same namespace. I could only move `StatusBarItemOptions` out to  `vscode`.